helpers/github.ts

```diff
@@ -383,6 +383,9 @@ export async function createDevPoolIssue(projectIssue: GitHubIssue, projectUrl:
   // if issue doesn't have the "Price" label then skip it, no need to pollute repo with draft issues
   if (!(projectIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.PRICE))) return;
 
+  // if the issue is opened by someone other than "ubiquibot" or "UbiquityOS", skip it
+  if (!authorizedUsers.includes(projectIssue.user?.login as string)) return;
```

Comment:
This doesn't make sense to me. Please understand that this function is for creating a new issue in the directory. I presume that `projectIssue` is from the other repository. So now no issues would be populated in the directory because the source issues are always created by someone other than our bots.

Reply:
You are right. Instead of doing the check on `projectIssue`, I'll do so on `createdIssue` since at that point the issue is created by our bot. Think this approach makes more sense right?